main.py

```diff
@@ -20,7 +20,7 @@
   development phases. This script represents the backbone of this application. 
   This script is called as below:
 
-  usage: main.py [-h] [--version] [--input FILEPATH] [--policy FILENAME] COMMAND
+  usage: main.py [-h] [--version] [--input FILEPATH] [--policy FILENAME] [--command_policy FILENAME] COMMAND
```

Comment:
Originally I was thinking that we would keep it in default_policy.json file also, but what happens is if you want to create multiple command files for your applications then you need to create multiple policy.files.  

So if there was another change, unrelated to the policy, like a new keyword to be added (ex:  "ok") then you would have to edit these additional policy files (which may be creating a burden on developers) so that they all contain the new ok keyword in 1 file versus multiple files.